src/EnergyPlus/HeatBalanceSurfaceManager.hh

```diff
@@ -64,6 +64,8 @@
 
 // EnergyPlus Headers
 #include <EnergyPlus.hh>
+#include <DataSurfaces.hh>
+#include <DataHeatBalance.hh>
```

Comment:
Could you explain this a little more? Everything I have tried results in errors.